lib/util/merge-confidence/index.spec.ts

```diff
@@ -0,0 +1,151 @@
+import * as httpMock from '../../../test/http-mock';
+import { getName } from '../../../test/util';
+import * as hostRules from '../host-rules';
+import {
+  getMergeConfidenceLevel,
+  isActiveConfidenceLevel,
+  satisfiesConfidenceLevel,
+} from '.';
+
+describe(getName(), () => {
+  describe('isActiveConfidenceLevel()', () => {
+    it('returns false if null', () => {
```

Comment:
```suggestion

    it('returns false if null', () => {
```
Can you add some newlines before each test for readability?